plugins/UFM_NDT_Plugin/ufm_sim_web_service/merger_resources.py

```diff
@@ -620,6 +624,151 @@ def post(self):
+    and send respond back
+    '''
+    def __init__(self):
+        super().__init__()
+
+    def get(self):
+        '''
+        Get cable validation report
+        '''
+        if not self.cable_validation_server_addr:
+            error_status_code = 400
+            error_response = "Failure: Cable validation server address not defined in config file."
+            logging.error(error_response)
+            return self.report_error(error_status_code, error_response)
+        # check if local request or remote and read credentials from config if need...
+        if self.cable_validation_server_addr in LOCAL_HOST_VALUES:
+            # local request - no need credentials
+            result, status_code = get_local_cable_validation_report(self.ufm_port)
+        else:
+            cv_username, cv_password = read_cv_credentials(self.cv_credentials_path)
+            if cv_username and cv_password:
+                result, status_code = get_cable_validation_report(self.cable_validation_server_addr,
+                                           self.cable_validation_request_port,
+                                           cv_username, cv_password)
+            else:
+                return self.report_error(status_code, "Failed to read CV credentials.")
+        if result:
+            return self.report_success(result)
+        else:
+            return self.report_error(status_code, "No cable validation report data was retrieved")
+
+    def post(self):
+        '''
+        Post rest 
+        '''
+        return self.report_error(405, "Method is not allowed")
+
+
+class MergerCableValidationEnabled(UFMResource):
+    '''
+    If cable validation server address set
+    and send respond back
+    '''
+    def __init__(self):
+        super().__init__()
+
+    def get(self):
+        '''
+        Get rest 
+        '''
+        if self.cable_validation_server_addr:
+            return self.report_success()
+        else:
+            return self.report_error(400, "Cable validation is not enabled.")
+
+    def post(self):
+        '''
+        Post rest 
+        '''
+        return self.report_error(405, "Method is not allowed")
+
+
+class MergerCableValidationConnectionCfg(UFMResource):
+    '''
+    Return cable validation status:
+    if it is running locally or on remote server and
+    if on remote ser5ver - if it has credentials and other stuff for connection
+    establishment
+    '''
+    def __init__(self):
+        super().__init__()
+
+    def get(self):
+        '''
+        Send respond with connectivity parameters
+        '''
+        logging.info("GET /plugin/ndt/cable_validation_connection_configuration")
+        # read credentials and info from credential file
+        # read credentials
+        # check config
+        responce_dict = {}
+        if not self.cable_validation_server_addr: # no cv server defined
+            responce_dict["status"] = "disconnected"
```

Comment:
why do we have "status" ? "disconnected" / "connected" 
it is confusing, the NDT merger is not managing the connection status.
upon request, the call to the cable validation will be done...I don't see a meaning here for the status.

Reply:
changed to is_enabled with True or False